routes.php

```diff
@@ -22,6 +22,10 @@
         return;
     }
 
+    if (Config::get('rainlab.translate::disableLocalePrefixRoutes', false)) {
```

Comment:
I think this can be moved on top of this function to prevent useless calls for App and Translator.